KD_Lib/Quantization/common/base_class.py

```diff
@@ -0,0 +1,74 @@
+import torch
+import os
+import time
+
+
+class Quantizer:
+    """
+    Baisc Implementation of Quantization for PyTorch models.
+    :param model (torch.nn.Module): Model that needs to be pruned
+    """
+
+    def __init__(self, model):
+        self.model = model
+        self.quantized_model = model
+
+    def quantize(self):
+        """
+		Function used for quantization
+		"""
+        raise NotImplementedError
+
+    def compare_model_sizes(self):
+        """
```

Comment:
a better name would be `get_model_sizes()` from users perspective

Reply:
I thought of the same previously. However, named it this way coz it will be used for comparison. I'll change it though, `get_model_sizes` seems nicer